feg/gateway/policydb/flow_parser.go

```diff
@@ -150,18 +154,24 @@ func parseAddress(addr string) (*address, error) {
 		return nil, fmt.Errorf("Invalid format for address %s", addr)
 	}
 	ipAddr := matches[0]
+	version := protos.IPAddress_IPV4
```

Comment:
why is this variable called version?  isn't it the actual ipv4 address?

Reply:
ipAddr is the actual address. version is IPAddress_IPV4 or IPAddress_IPV6